packages/extension/src/view/devtools/components/tabContent/cookies/cookiesListing/index.tsx

```diff
@@ -38,17 +38,16 @@ const CookiesListing = () => {
   const filteredCookies = useFilterManagementStore(
     ({ state }) => state.filteredCookies
   );
+  const cookiesAvailable = useFilterManagementStore(
```

Comment:
Had to get the `cookiesAvailable` separately because combining it with filteredCookies made the whole component go into an infinite loop.